ajax/libs/plotly.js/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "plotly.js",
+  "version": "1.28.3",
+  "description": "The open source javascript graphing library that powers plotly",
+  "filename": "plotly.min.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/plotly/plotly.js.git"
+  },
+  "author": "Plotly, Inc.",
+  "homepage": "https://plot.ly/javascript/",
+  "keywords": [
+    "graphing",
+    "plotting",
+    "data",
+    "visualization",
+    "plotly"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/plotly/plotly.js.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "plotly!(*with-meta)*"
```

Comment:
Don't we need to exclude `.md`, `.json` and so on?

Reply:
@sufuf3 no response here?

Reply:
I responded here https://github.com/cdnjs/cdnjs/pull/11504#issuecomment-327064989